### PR TITLE
Lowercase atom

### PR DESCRIPTION
**Pull request by chulkilee**
Based on the discussion in #154

### CODE REVIEW

File: apps/opentelemetry/include/otel_span.hrl [hunk 1]
```diff
@@ -42,8 +42,8 @@
           name                    :: unicode:unicode_binary() | atom(),
 
           %% Distinguishes between spans generated in a particular context. For example,
-          %% two spans with the same name may be distinguished using `CLIENT` (caller)
-          %% and `SERVER` (callee) to identify queueing latency associated with the span.status
+          %% two spans with the same name may be distinguished using `client` (caller)
```
Comment (chulkilee) [inline apps/opentelemetry/include/otel_span.hrl:45]: Is it better to keep the value in the spec here? Or is it better to change it to actual representation in this library? It applies to all changes in docs/comments.

Comment (tsloughter) [reply apps/opentelemetry/include/otel_span.hrl:45]: Sorry, only now noticed you had a question in this PR. I'd say change it to the actual representation, so lowercase.

**Comment by tsloughter** [comment]
@chulkilee can you rebase this and mark it as ready? It'd be great to get this merged soon since it'll require updates to the integrations.

**Comment by tsloughter** [comment]
Ugh, looks like EasyCLA is busted and can't merge without that succeeding.

**Review by tsloughter — APPROVED** [review]

**Comment by tsloughter** [comment]
easyCLA is stuck so I'm closing this PR to reopen it.

**Pull request merged**